test/libgit2.jl

```diff
@@ -219,6 +219,13 @@ mktempdir() do dir
                 @test LibGit2.iszero(commit_oid2)
                 commit_oid2 = LibGit2.commit(repo, commit_msg2; author=test_sig, committer=test_sig)
                 @test !LibGit2.iszero(commit_oid2)
+                auths = LibGit2.authors(repo)
+                @test length(auths) == 3
```

Comment:
which repo is this, is it constructed from scratch in this test harness, or is it referring to something like Example.jl that may change over time?